ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -33,4 +33,4 @@ if [ ${FIPS_ENABLED} = "true" ]; then
 	cat >> "${out}" << EOF
 fips: true
 EOF
-fi
\ No newline at end of file
+fi
```

Comment:
LOL this change made us rehearse a million non-GCP jobs :trollface:

Reply:
Ups.. I wanted to reset this one, but my vim is too smart I guess :)

Reply:
Not a big deal :)